app/services/translations.ts

```diff
@@ -0,0 +1,81 @@
+// Vendor
+import FastBoot from 'ember-cli-fastboot/services/fastboot';
+import fetch from 'fetch';
+import IntlService from 'ember-intl/services/intl';
+import Service, {inject as service} from '@ember/service';
+import Shoebox from 'ember-boilerplate/services/shoebox';
+
+// Config
+import config from 'ember-boilerplate/config/environment';
+
+export type AvailableLocale = 'en-ca';
+
+const pathForLocale: Record<AvailableLocale, string> = {
+  'en-ca': '/assets/translations/en-ca.json'
```

Comment:
Does it mean we're going to have to update this file if a project is bilingual or the main locale if `fr-ca` ?

Reply:
Yep!